### PR TITLE
fixed incorrect numbering on symptom checker

### PR DESCRIPTION
**Pull request by omarzoghayyer**
### An Overview of the Changes
Added missing currentStep -1 and totalSteps -1 in the SymptomsCheck.js file.  Both were missing in two different location of the file.

### CODE REVIEW

File: client/src/pages/OfferHelp.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React, { useState } from "react";
 import { withRouter, Link } from "react-router-dom";
-
```
Comment (omarzoghayyer) [inline client/src/pages/OfferHelp.js:3]: I added a link by mistake, then deleted it. Please Ignore.

**Comment by indirabhowmick** [comment]
@vikjoshi - PLease test the changes so that the PR can be moved to done column

**Comment by vikjoshi** [comment]
@omarzoghayyer @anniegiang, checked on staging- one more update to make. We only have 8 questions and right now the numbering goes up to 9 (e.g., Question 1/9, 2/9, 3/9 etc.)- can we change to 8? 

image

**Comment by omarzoghayyer** [comment]
@vikjoshi For sure. Ill work on it now

**Comment by indirabhowmick** [comment]
@omarzoghayyer - Can you please move the ticket back to In Progress status?

**Comment by omarzoghayyer** [comment]
@indirabhowmick This ticket has been merged into master

**Comment by indirabhowmick** [comment]
@omarzoghayyer - Thanks for the update!
@vikjoshi - Please test the changes when you get a chance? Once tested we can move the ticket to Done

**Comment by vikjoshi** [comment]
@omarzoghayyer, took a look and still seeing this issue on staging. 
image

**Comment by omarzoghayyer** [comment]
Thats odd. Ill take a look at it now

Screen Shot 2020-05-20 at 3 08 45 PM

**Comment by omarzoghayyer** [comment]
@vikjoshi Im not sure what happened. I looked at the code and it seems not to be changed. Ill go ahead and make a pull request for the new changes.

**Comment by omarzoghayyer** [comment]
Sorry for the confusion @vikjoshi The changes for the numbering issue was resolved in a different PR check this https://github.com/FightPandemics/FightPandemics/pull/396/files

This PR was intended to solve the missing currentStep -1 and totalSteps -1 in the SymptomsCheck.js file. Both were missing in two different location of the file.

**Comment by vikjoshi** [comment]
no problem @omarzoghayyer - on staging in Desktop Web, I'm still not seeing the change. Should I follow up with whoever was working on https://github.com/FightPandemics/FightPandemics/pull/396/files

**Pull request merged**